ui/src/features/Editor/components/OverlayUI/components/Toolbox/index.tsx

```
@@ -163,35 +163,52 @@ const Toolbox: React.FC<Props> = ({
           ) : (
-            <IconButton
-              key={tool.id}
-              className={`dndnode-${tool.id} rounded-[4px]`}
-              tooltipPosition="right"
-              tooltipText={tool.name}
-              icon={tool.icon}
-              onDragStart={(event) => onDragStart(event, tool.id)}
-              draggable
-              disabled={tool.disabled}
-            />
+            <div className="bg-secondary rounded-md">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)